examples/IBFE/explicit/ex5/example.cpp

```diff
@@ -529,17 +536,15 @@ run_example(int argc, char** argv)
         // velocity.
         if (SAMRAI_MPI::getRank() == 0)
         {
-            drag_stream.open("C_D.curve", ios_base::out | ios_base::trunc);
-            lift_stream.open("C_L.curve", ios_base::out | ios_base::trunc);
-            U_L1_norm_stream.open("U_L1.curve", ios_base::out | ios_base::trunc);
```

Comment:
Please do not remove output streams or change the names of existing output streams.

Reply:
Should C_D and C_L be computed from the exterior traction then? I had output streams of the force coefficients using both forces and fluid traction. Also U_L1 and U_L2 are not being used anywhere in the code. That's why I removed them.

Reply:
OK, if they are not being used in the original example, then it is definitely a good idea to remove them.

I would set this up to use the forces to compute C_D and C_L, regardless of the method configuration. Then, if it is possible to extract the traction accurately, there could be additional streams that are activated.